sync_issues_to_jira/sync_issue.py

```diff
@@ -67,6 +67,14 @@ def handle_issue_labeled(jira, event):
 
     labels = list(jira_issue.fields.labels)
     new_label = _get_jira_label(event["label"])
+
+    # ignore labels that start with "Status:" and "Resolution:"
+    # these labels are mirrored from Jira issue
+    # and should not be mirrored back as labels
+    ignore_prefix = ("status:", "resolution:")
+    if new_label.lower().startswith(ignore_prefix):
+        return
```

Comment:
It's a bit borderline for 3 lines of duplicate code, but what about making a function `def is_ignored_github_label(labelname)` and then call it from both places?